src/components/Dashboard/PrintInfo.js

```diff
@@ -33,9 +21,24 @@ const PrintInfo = ({ isLayout }) => {
                 {`${i18n.t('Print Preview')}: ${infoHeader}`}
             </p>
             <div style={{ maxWidth }}>
-                <p
-                    className={classes.info}
-                >{`${infoPartOne} ${infoPartTwo}`}</p>
+                <p className={classes.info}>
+                    {i18n.t('For best print results:')}
+                </p>
+                <p className={classes.info}>
+                    <ul className={classes.infoList}>
+                        <li>{i18n.t('Use Chrome or Edge web browser')}</li>
+                        <li>
+                            {i18n.t(
+                                'Wait for all dashboard items to load before printing this page'
```

Comment:
Perhaps omit the last two words `this page` since the dashboard can have multiple pages.

Reply:
@cooper-joe